documentation/source/user_section/tutorials/other/spinalcord-smoothing.rst

```diff
@@ -0,0 +1,34 @@
+.. TODO:
+
+   Is this one-page tutorial necessary? It is basically just telling users that the ``sct_smooth_spinalcord`` tool exists. (Compared to other tutorials, which demonstrate multi-step workflows.)
+
+   So, I am thinking that maybe this page will be unnecessary once we organize the "Command-Line Tools" page into one-page-per-script. We could simply have all of this information on the dedicated "sct_smooth_spinalcord" page instead, and save the "Tutorials" for complex workflows only.
+
+.. _spinalcord-smoothing:
+
+Spinal cord smoothing as a preprocessing operation
+##################################################
+
+SCT provides a spinal cord smoothing function that takes into account the curvature of the cord, preserving the boundary between the spinal cord and cerebrospinal fluid. This is useful in a variety of situations:
+
+* It can be used to improve sensitivity of fMRI results while minimizing contamination from CSF
+* It can also be used to obtain more reliable cord segmentations, because smoothing will sharpen the edge of the cord and will blur out possible artifacts at the cord/CSF interface.
+
+.. code::
+
+   sct_smooth_spinalcord -i t1.nii.gz -s t1_seg.nii.gz
+
+:Input arguments:
+   - ``-i`` : The input image.
+   - ``-s`` : A spinal cord segmentation mask corresponding to the input image. This is needed as ``sct_smooth_spinalcord`` performs a 1D smoothing operation following the cord centerline (as opposed to the cortical surface smoothing in FreeSurfer, which is 2D).
+
+:Output files/folders:
+   - ``t1_smooth.nii`` : The input image, smoothed along the spinal cord. (TODO: Why are we outputting the uncompressed ``.nii`` file here?
```

Comment:
Possible bug in `sct_smooth_spinalcord`? (1/2)

Reply:
ah! yes, thank you for catching that

Reply:
--> #3503